driver-sync/src/test/functional/com/mongodb/client/CrudProseTest.java

```diff
@@ -594,4 +596,21 @@ public int getV() {
     private interface TriConsumer<A1, A2, A3> {
         void accept(A1 a1, A2 a2, A3 a3);
     }
+
+    /**
+     * This method is used instead of {@link ClientSession#withTransaction(TransactionBody)}
+     * because reactive {@link com.mongodb.reactivestreams.client.ClientSession} do not support it.
```

Comment:
A documentation link from `driver-sync` to a program element located in `driver-reactive-streams` does not work and is displayed by the IDE as an error, so let's not use it:

```suggestion
     * because reactive {@code com.mongodb.reactivestreams.client.ClientSession} do not support it.
```